pkg/apb/bind.go

```diff
@@ -25,17 +25,13 @@ func Bind(
 	log.Notice(fmt.Sprintf("ServiceInstance.Description: %s", instance.Spec.Description))
 	log.Notice("============================================================")
 
-	var client *Client
-	var err error
-
-	if client, err = NewClient(log); err != nil {
-		return "", nil, err
-	}
-
-	podName, err := client.RunImage("bind", clusterConfig, instance.Spec, instance.Context, parameters)
+	podName, err := ExecuteApb(
+		"bind", clusterConfig, instance.Spec,
+		instance.Context, parameters, log,
+	)
```

Comment:
+1, I was musing over this earlier. Should probably define a finite amount of typed actions ExecuteApb actually accepts so you can't build a broker that feeds garbage into ExecuteApb.